### ISSUE TITLE
Divo Hangs When Unable to Pull Image

### ISSUE THREAD

**Issue by JarredOlson** [smartcitiesdata/divo#11]
It would be nice if Divo would timeout and/or give a useful error message when it is unable to pull images.

**Comment by ManApart** [comment]
It looks like when docker compose runs, if it fails to pull an image, it returns an empty message, and divo continues even though an exit code of 1 is returned.

Adding `stderr_to_stdout: true)` returns the error message. Divo needs to report this and stop processing.

```
iex(5)> a = System.cmd("docker-compose", ["up"], stderr_to_stdout: true)  
{"Pulling presto (testo:latest)...
time=\"2019-06-04T16:06:53-04:00\" level=error msg=\"Error retrieving credentials\" error=\"ecr: Failed to get authorization token: NoCredentialProviders: no valid providers in chain. Deprecated.
For verbose messaging see aws.Config.CredentialsChainVerboseErrors\"
Get https://dkr.ecr.us-east-2.amazonaws.com/v2/presto-testo/manifests/latest: no basic auth credentials", 
 1}
```

**Comment by ManApart** [comment]
Fixed in 1.1.6

```
  2) PrestoIngrationTest: failure on setup_all callback, all tests have been invalidated
     ** (RuntimeError) Docker Compose exited with code: 1. Pulling presto (dkr.ecr.us-east-2.amazonaws.com/presto-testo:latest)...
     time="2019-06-05T09:52:51-04:00" level=error msg="Error retrieving credentials" error="ecr: Failed to get authorization token: NoCredentialProviders: no valid providers in chain. Deprecated.\n\tFor verbose messaging see aws.Config.CredentialsChainVerboseErrors"
     Get https://dkr.ecr.us-east-2.amazonaws.com/v2/presto-testo/manifests/latest: no basic auth credentials

     stacktrace:
       lib/divo/compose.ex:66: Divo.Compose.log_compose/1
       lib/divo/compose.ex:22: Divo.Compose.run/1
       test/integration/data/presto_integration_test.exs:3: PrestoIngrationTest.__ex_unit_setup_all_0/1
       test/integration/discovery_api/data/presto_integration_test.exs:1:PrestoIngrationTest.__ex_unit__/2

Finished in 42.2 seconds
2 tests, 0 failures, 2 invalid
```
